src/gui/rss/automatedrssdownloader.ui

```diff
@@ -424,7 +424,17 @@ Supports the formats: S01E01, 1x1, 2017.12.31 and 31.12.2017 (Date formats also
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
-     <item>
+      <item>
+      <widget class="QPushButton" name="editTitleBtn">
+        <property name="enabled">
+         <bool>true</bool>
+        </property>
+        <property name="text">
+         <string>&amp;Edit rule title</string>
```

Comment:
There is a context menu action "Rename rule..." 
also "E" is already used for "Export..."

Reply:
> Rename rule...

:+1:

Reply:
Oof okay, that was really stupid of me. I even saw the hotkey for the others but it somehow slipped through. Sorry! I'm trying to fix the other PR and then will fix this one ASAP too, sorry guys!